tools/sz_repo_cli/lib/src/commands/src/test_command.dart

```diff
@@ -7,14 +7,21 @@
 // SPDX-License-Identifier: EUPL-1.2
 
 import 'dart:async';
+import 'dart:io';
 
 import 'package:args/command_runner.dart';
-
 import 'package:sz_repo_cli/src/common/common.dart';
 
 class TestCommand extends Command {
-  TestCommand(this._repo) {
+  TestCommand(this.repo) {
     argParser
+      ..addOption(
+        maxConcurrentPackagesOptionName,
+        abbr: 'c',
+        defaultsTo: '5',
+        help:
+            'How many packages at most should be processed at once. Helpful for a CI or not as powerful PCs to not have so much processing at once.',
```

Comment:
Should we also mention that 0 or negative would result into processing all at once?